### ISSUE TITLE
/play no longer working

### ISSUE THREAD

**Issue by USAMcDoanldsUSA** [codetheweb/muse#945]
**What's happening?**
When I try to use the /play command, the bot responds with "Invalid regular expression: missing /". I suspect this is a ytdl issue but I figured I would ask. 

**Logs**
I do not receive any additional logs when running docker logs on the container

**Runtime**
I'm running Muse:
- [ ] Directly from the cloned repository
- [x] Inside a Docker container
- [ ] Something else (please elaborate)

**Versions**
- Muse: latest
- Docker (if applicable): Docker version 23.0.1, build a5ee5b1
- OS: Red Hat Enterprise Linux release 8.6 (Ootpa)

**Comment by cgjeffries** [comment]
Seems related to https://github.com/fent/node-ytdl-core/issues/1227

**Comment by USAMcDoanldsUSA** [comment]
Yep. Seems to be working again if I downgrade ytdl-core to 4.10.0.

**Comment by Toarulle** [comment]
> Yep. Seems to be working again if I downgrade ytdl-core to 4.10.0.

Having the same issue. Though I am running the docker through unRAID and seems to be directly from the repository. I can therefore not downgrade any specific software myself. I think anyways. Quite new to this stuff.

I'd gladly take any help

**Comment by crazy4ever** [comment]
> > Yep. Seems to be working again if I downgrade ytdl-core to 4.10.0.
> 
> Having the same issue. Though I am running the docker through unRAID and seems to be directly from the repository. I can therefore not downgrade any specific software myself. I think anyways. Quite new to this stuff.
> 
> I'd gladly take any help

If you are unraid follow the steps below to downgrade ytdl-core to 4.10.0:

1. Open Unraid
2. Docker tab
3. Click "Muse" container icon
4. Edit
5. Basic View (Top right)
6. To the "Post Arguments" add the rule: 

tini -- sh -c "npm install ytdl-core@4.10.0 && yarn start"

7. Apply

This fixes the bot breaking down, and if the Container gets an update with a ytdl-core fix just remove the line, apply, update afterwards and all should be good.

**Comment by shiftybitshiftr** [comment]
Closing this as part of the support backlog cleanup. This report has not been updated in over a year and predates the current playback/voice/runtime stack. Please open a fresh issue if it still reproduces on v2.11.3 or newer, and include the startup `Running version ...` line, image tag/digest or install method, host architecture, the exact URL/query/command, and fresh logs.
